start.xenit.eu/src/main/java/eu/xenit/alfred/initializr/start/extensions/alfred/telemetry/grafana/GrafanaDockerComposeProjectGenerationConfiguration.java

```diff
@@ -22,7 +22,7 @@
     DockerComposeCustomizer grafanaDockerComposeCustomizer() {
         return (compose) -> compose.file("grafana").getServices()
                 .service("grafana")
-                .image("hub.xenit.eu/grafana:5.0")
+                .image("hub.xenit.eu/public/grafana:5.0")
```

Comment:
Does this image still exists ? I think we need to create a ticket for that first ?

Reply:
nope, my bad
`hub.xenit.eu/public/grafana:0.0.1-5` seems to exist. What about that one?

Reply:
@driesdeproost `5:0` also does not exist
1. you should look that up first
2. you should verify this works

Reply:
for point 2 however, there seems to be a bug with the application of our amp plugin in a setup with telemetry and micrometer.

Out of scope for this ticket though

Reply:
Aha :+1: